scripts/book.js

```diff
@@ -0,0 +1,19 @@
+// Description:
+//  Script to confirm rooms bookings
+//
```

Comment:
```suggestion
//
// Configuration:
// CALENDAR_CLIENT_ID - calendar oauth client id
// CALENDAR_CLIENT_SECRET - calendar oauth secret
// CALENDAR_OAUTH_ROOM_ID - room of a person who can grant access to bots calendar
//
```

I think those properties are required here? 🙂